README.md

```diff
@@ -765,3 +765,18 @@ Removing ccloudvm service
 ```
 
 Once you run ccloudvm teardown, ccloudvm will be unusable until you run ccloudvm setup.
+
+## Misc
+
+### bash completion
+The script that might help with auto completion is located [here](./scripts/ccloudvm_bashcomplete). This can be either placed at `/etc/bash_completion.d/` of your machine or as described below.
+
+```
+$ mkdir ~/.bash_completion.d
+$ scp scripts/ccloudvm_bashcomplete ~/.bash_completion.d/ccloudvm
```

Comment:
should this be cp rather than scp?

Reply:
Yeah.. should be cp. Or I can make them curl it from the github url. Leaning towards the github url. WDYT?

Reply:
Either is fine for me.  The curl command would be useful if we were to ever do a binary distribution but this is unlikely so cp is fine.